<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_widget</class>
 <widget class="QDialog" name="dialog_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>191</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Sans Mono</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Choose a theme</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="radio_layout">
     <item>
      <widget class="QRadioButton" name="system_radio">
       <property name="text">
        <string>System theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="light_radio">
       <property name="text">
        <string>Light theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dark_radio">
       <property name="text">
        <string>Gray theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="very_dark_radio">
       <property name="text">
        <string>Dark theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="military_green_radio">
       <property name="text">
        <string>Military theme</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>dialog_widget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>dialog_widget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
